src/drive/mobile/styles/main.styl

```diff
@@ -18,7 +18,6 @@ html {
 }
 
 :global
-    [role=banner] .coz-bar-btn.coz-bar-burger{
-        visibility hidden
-        flex 0 0 .22em
+    [role=banner] .coz-drawer--settings > div > :nth-child(-n+3) {
```

Comment:
Lol. Do you mind adding a comment saying what this does please? Or making the selector more explicit^^